.github/CODEOWNERS

```diff
@@ -41,6 +41,7 @@
 */brave*              @AdmiringWorm
 */nexus-repository*   @chocolatey-community/chocolatey-team-maintainers
 */totalcommander*     @pauby
+*/kubescape*          @AdmiringWorm
```

Comment:
I am **NOT** the maintainer of this package, nor have I agreed to be one.

The maintainer is typically the one that submits the PR.

Reply:
Okay. The reason why I didn't add myself to the CODEOWNERS is that I don't have write access to this repo and this is causing an error in that file unless you actually give me write access:

![image](https://github.com/chocolatey-community/chocolatey-packages/assets/43995067/3e55c7d4-517e-4dc5-b6b9-ccbf2fb2aad2)

 Since you approved the migrating request, I added you here. sorry about that.

Reply:
I am aware of the error regarding write access.

The intention of being added in the CODEOWNERS is so we know who to reference manually to have a look over any PRs created and to implement some automation for it in the future.

Thanks for updating.